docs/contributing/assets/allowed_actions.json

```diff
@@ -22,7 +22,9 @@
         "a4f60bb28d35aeee14e6880718e0c85ff1882e64",
         "9d1e0624a798bb64f6c3cea93db47765312263dc",
         "3cfe3a4abbb849e10058ce4af15d205b6da42804",
-        "v4.0.0"
+        "v4.0.0",
+        "v6.0.1",
+        "a4f60bb28d35aeee14e6880718e0c85ff1882e64"
```

Comment:
The version in line 27 is already there (see line 22) and must not be added a second time.